src/elements/OcSidebarNavItem.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+  <li :class="active">
+    <router-link :to="target">
+      <span class="uk-margin-small-right">
+        <oc-icon variation="inverted" :name="icon" />
```

Comment:
https://getuikit.com/docs/text#vertical-alignment might help to fix alignment of icon - helped for @butonic I think